src/applications/facility-locator/sass/facility-locator.scss

```diff
@@ -11,12 +11,11 @@ $facility-locator-color-gray: #ccc;
 $facility-locator-shadow: rgba(0, 0, 0, 0.5);
 
 .facility-locator {
-
-  @media(max-width: $large-screen) {
+  @media (max-width: $large-screen) {
```

Comment:
All of the changes to this file except the additions at the end of the file are formatting from prettier.